holesky/README.md

```diff
@@ -77,11 +73,19 @@ In order to opt-in into EigenDA as an Operator, you must meet the following dele
 - Have more than 1.1x current lowest-stake Operator in the active Operator set. Please see [EigenDA Churn Approver](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) for more detail.
 - The operator to churn out has less than 10.01% of the total stake
 
-Execute the following command to opt-in to EigenDA AVS:
+Execute the following command based on the quorum you want to opt-in in EigenDA AVS:
 
 ```
-./run.sh opt-in
+./run.sh opt-in <quorum>
+
+# for opting in to quorum 0:
+./run.sh opt-in 0
+
+# for opting in to quorum 0 and 1:
+./run.sh opt-in 0,1 
+
 ```
+You only need to provide the quorum which you want to opt into. For example if you are already registered to quorum `0` and want to opt-in one more quorum `1`, then you just need to set `<quorum>` as `1` while opting in again.
 
 :::warn
 Operator must wait up to 6 hours if the delegation happened after you opt-in to the EigenDA AVS. EigenLayer's AVS-Sync component runs at 6 hour batch intervals to update the delegation totals on chain for each operator. If you are unable to opt in despite having sufficient delegated stake, please wait at least 6 hours, then retry opt-in.
```

Comment:
@shrimalmadhur  can we fix this ":::warn" to ":::warning"?

I don't think you introduced this error, but it would fix the formatting.